### DISCUSSION TITLE
Piecewise Domain

### DISCUSSION THREAD

**Discussion by joafigue** [zwicker-group/py-pde#234] [Q&A] [answered]
Hi, 
I'm considering using this package as it does almost everything I need, but I need a critical feature that does not seem to be possible.

I need to solve Laplace equation through multiple connected Cartesian domains (With each properly defined boundary conditions), in a sense is similar to [domain with a hole](https://github.com/zwicker-group/py-pde/discussions/142), but maybe my use case is different enough that is possible to do it.

```
                         +---------------+ 
                         |               | 
   +---------------------+               | 
   |                     |    D2         | 
   |    D1               |               | 
   |                     |               | 
   +---------------------+               | 
                         |               +---------------+
                         |               |               |
                         |               |    D3         |
                         |               |               |
                         |               |               |
                         +---------------+---------------+
```

I have been trying to work-around the issue, with no real success so I'm asking here. Thanks

**Comment by david-zwicker** [comment, marked as answer]
I'm afraid that this configuration is already too complicated for the package.

**Reply by joafigue** [reply]
Thanks for the answer.
It is unfortunate, but it's good to have confirmation.
